src/Views/Registration/Registration.js

```diff
@@ -1,11 +1,100 @@
-import React from 'react'
-
-const Registration  = () => {
-    return ( 
-        <div>
-            Registartion
-        </div>
-      );
-}
- 
-export default Registration;
\ No newline at end of file
+import { useState } from "react";
+import { useHistory } from "react-router-dom"
```

Comment:
what about `import React from 'react'`?

Reply:
It does work, probably because React is imported in index or App.js